helper/resource/testing.go

```diff
@@ -633,6 +641,12 @@ type TestStep struct {
 	// ID of that resource.
 	ImportState bool
 
+	ImportStateKind ImportStateKind // or ImportStateStrategy or ImportStateSubmode or ImportStateFlavor or ...
+
+	// ImportStateBlockConfig, if non-empty, supplies declarative import
+	// configuration. This is (?mutually exclusive of ImportStateID + ResourceName?).
+	ImportStateBlockConfig string
```

Comment:
If we can generate config on our first try, then the usefulness of this is secondary.